ui-tests/src/test/java/org/openmrs/reference/page/EditVisitNotePage.java

```diff
@@ -10,15 +10,28 @@
 package org.openmrs.reference.page;
 
 import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
 
 public class EditVisitNotePage extends VisitNotePage {
-
+	
+    private static final By NOTE = By.id("w10");
+    private static String  note = "this is edited note";
```

Comment:
Should `note` be not a final  variable ?

Reply:
Resolved thanks

Reply:
Hello @kdaud  Any more reviews and recommendations

Reply:
@sherrif10 what do you say about this artifacts [here](https://stackoverflow.com/questions/7259687/java-naming-convention-for-static-final-variables) about naming convention of static final variables ?